Formula/duktape.rb

```diff
@@ -0,0 +1,38 @@
+class Duktape < Formula
+  desc "Embeddable Javascript engine with compact footprint"
+  homepage "http://duktape.org"
+  url "http://duktape.org/duktape-2.2.1.tar.xz"
+  sha256 "3abe2eed2553305262b892c98f550bb1a94cf4fd73b51dc5c176fe08e7ade7f2"
+
+  def install
+    inreplace "Makefile.sharedlibrary" do |s|
```

Comment:
The need for this patch is a little unfortunate. Seems like this could be handled by the `tools/configure.py` or something else in the main configure/install process?

Reply:
Upstream PR: https://github.com/svaarala/duktape/pull/1922

Reply:
:+1: